cmd/vic-machine/create/create.go

```diff
@@ -1285,12 +1289,31 @@ func logArguments(cliContext *cli.Context) {
 			continue
 		}
 
+		s := cliContext.String(f)
+		if s != "" {
+			flag := fmt.Sprintf("--%s=%s", f, s)
+			log.Debug(flag)
+			args = append(args, flag)
+			continue
+		}
+
+		b := cliContext.Bool(f)
+		bT := cliContext.BoolT(f)
+		if b && !bT {
+			flag := fmt.Sprintf("--%s=%t", f, true)
+			log.Debug(flag)
+			args = append(args, flag)
+			continue
+		}
+
 		match = func() (result bool) {
 			result = false
```

Comment:
I would restructure this function a little bit. I am little suspicion about presence of recover() and usage of result variable. Not sure recover() is really needed here.

```
match = func() boot {
    is := cliContext.IntSlice(f)
    if is != nil {
        ...
    }
    return is != nil
}
```